src/wp-content/themes/classicpress-twentyfifteen/functions.php

```diff
@@ -22,14 +22,14 @@ function cp2015_setup() {
 add_action( 'wp_enqueue_scripts', 'cp2015_enqueue_styles' );
 
 function cp2015_enqueue_styles() {
-
-    $parent_style = 'twentyfifteen-style';
-
-    wp_enqueue_style( $parent_style, get_template_directory_uri() . '/style.css' );
-    wp_enqueue_style( 'classicpress-twentyfifteen',
-        get_stylesheet_directory_uri() . '/style.css',
-        array( $parent_style ),
-        filemtime( get_stylesheet_directory() . '/style.css' )
-    );
+	
+	$parent_style = 'twentyfifteen-style';
+	
+	wp_enqueue_style( $parent_style, get_template_directory_uri() . '/style.css' );
+	wp_enqueue_style( 'classicpress-twentyfifteen',
+			 get_stylesheet_directory_uri() . '/style.css',
+			 array( $parent_style ),
+			 classicpress_asset_version( 'style', 'classicpress-twentyfifteen' );
```

Comment:
Syntax error on this line (extra semicolon).  Please test your code.

Reply:
I fixed this in commit db13335, and also made some further fixes to the code indentation.  In the future I'd recommend using an editor that can show you the difference between tabs and spaces, and reading over https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#indentation.